src/Components/FitnessFilters.js

```diff
@@ -93,6 +107,9 @@ class FitnessFilters extends React.Component {
       <View style={filterStyle.wrapper}>
         <Text style={filterStyle.counter}>{this.state.displayedDrills.length} drills available</Text>
         <ScrollView contentContainerStyle={filterStyle.filters}>
+          <View style={filterStyle.filter}>
+            <Button title="Favorite Drills" onPress={() => this.onFavoritesChange()} active={selectedFavorites} />
```

Comment:
The rendering looks a bit like a button to open something

Maybe we could use a checkbox ?
And a wording more like "My Favorites only"?

Reply:
Done with wording change, we can change to checkbox later on depending on feedback